CHANGELOG.md

```diff
@@ -6,454 +6,494 @@ All notable changes to this project will be documented in this file.
-  - Containers: Settings for proxy environment variables
-- User Interface
-  - Added mandatory Subscription field to Microsoft Azure Discovery screen
-  - Added Notifications Drawer and Toast Notifications List
-  - Added support for vSphere Distributed Switches
-  - Added support to show child/parent relations of Orchestration Stacks
-  - Added Middleware Messaging entities to topology chart
-  - I18n support for UI plugins
-  - Arbitration Profiles management for Service Broker
-  - Re-check Authentication button added to Provider list views
-  - Provisioning button added to the Templates & Images list and summary screens
-  - Subtype option added to Generic Catalog Items
-  - About modal added to OPS UI
-  - Both UIs updated to latest PatternFly and Angular PatternFly
-  - Self Service UI language selections separated from Operations UI
-  - Internationalization
-    - Virtual Columns
-    - Toolbars
-    - Removed string interpolation (for better localization)
-    - Changed to use gettext’s pluralization
-  - Ansible Tower Jobs moved to the Configuration tab (from Clouds/Stacks)
-  - Interactivity added to C3 charts on C&U screens  
-  - Settings moved to top right navigation header
-  - C3 Charts fully implemented - chart interaction coming soon!
-  - Tagging for Ansible Tower job templates
-  - Live Search added to bootstrap selects
-  - Self Service UI Order History with detail
-  -  Add GUID validation for certain Azure fields in Cloud Provider screen
-  - OpenStack: Register new Ironic nodes through Mistral
-  - Timelines resdesigned
-  - vSphere Distributed Switches tagging
-  - Patternfly Labels for OpenSCAP Results
-  - Operations UI Notifications
+  - UI for creating subnets
+- OpenStack
+  - Collect inventory for cloud volume backups
+  - Show topology for undercloud
+  - Associate/Disassociate Floating IPs
+  - Host Aggregates
+  - Region Support
+  - Cleanup of Nova services after scale down
+  - Prevent retired instances from starting
+  - Add hardware state for Hosts
+  - Map Flavors to Cloud Tenants during Openstack Cloud refresh
+  - Cinder backup/restore actions added to model
+  - UI to register Ironic nodes through Mistral
+- Red Hat Enterprise Virtualization
+  - Get Host OS version and type
+  - Disk Management in VM Reconfigure
+  - Snapshot support
+  - Report manufacturer and product info for RHEVM hosts
+- Storage
+  - New Swift Storage Manager
+  - New Cinder Storage Manager
+  - Initial User Interface support for Storage Managers
+- VMware vSphere
+  - Storage Profiles modeling and inventory
+  - Datastores are filtered by Storage Profiles in provisioning
+- vCloud
+  - Collect status of vCloud Orchestration Stacks
+  - Add Network Manager
+  - Collect networking inventory
+  - Cloud orchestration stack operation: Create and delete stack
+  -Collect virtual datacenters as availability zones
+  -Event Catcher
+
+### REST API
+
+- Token manager supports for web sockets
+- Added querying for cockpit support
```

Comment:
Probably related to https://github.com/ManageIQ/manageiq/pull/10871 with the introduction of the new decorator supports_cockpit? ie. GET /api/services?expand=vms&decorators=vms.supports_cockpit? 

Maybe could be rephrased as:
- Added ability to query if vms support cockpit